src/client/components/resize-handle/resize-handle.tsx

```diff
@@ -31,36 +34,21 @@ export interface ResizeHandleProps {
 
 export interface ResizeHandleState {
   dragging?: boolean;
-
   startValue?: number;
   currentValue?: number;
   anchor?: number;
 }
 
 export class ResizeHandle extends React.Component<ResizeHandleProps, ResizeHandleState> {
-  private offset = 0;
-
-  constructor(props: ResizeHandleProps) {
-    super(props);
-
-    this.state = {};
-
-    this.onGlobalMouseUp = this.onGlobalMouseUp.bind(this);
-    this.onGlobalMouseMove = this.onGlobalMouseMove.bind(this);
-  }
 
-  componentDidMount() {
-    this.setState({
-      currentValue: this.constrainValue(this.props.initialValue)
-    });
-  }
+  state: ResizeHandleState = {};
 
-  onMouseDown(event: MouseEvent) {
+  onMouseDown = (event: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
As you are not using any `div`-related properties I would advise to replace `HTMLDivElement` with `HTMLElement` (according to *Liskov substitution principle*)

Reply:
I agree but it's not related to LSP :)